src/main/flight/pid.c

```diff
@@ -284,6 +288,8 @@ static FAST_RAM float crashGyroThreshold;
 static FAST_RAM float crashSetpointThreshold;
 static FAST_RAM float crashLimitYaw;
 static FAST_RAM float itermLimit;
+FAST_RAM float throttle_boost;
```

Comment:
camelCase
`throttleBoost`, `throttleLpf` 

And PID filters are defined `static` before `pidInitFilters`

Reply:
(With LTO(-flto) , inlining is done over whole project, so penalty is quite small )

Reply:
Right. Well if you think wrapping that filter and the boost setting would be a material improvement in the scope of this project I can certainly do it. I'm not passionate about it either way.

Reply:
It won't help ... `throttle` handling needs major refactor. Then throttleBoost can be filter function, with reasonable interface/dependencies.